taca/storage/storage.py

```diff
@@ -64,6 +81,93 @@ def archive_to_swestore(days, run=None):
                 else:
                     LOG.info('No old runs to be archived')
 
+
+def cleanup_swestore(days,dry_run=False):
+    """
+        Remove archived runs from swestore
+
+        :param int days: Threshold days to check and remove
+    """
+    config = get_config()
+    LOG = get_logger()
+    runs = filesystem.list_runs_in_swestore(path=config.get('cleanup').get('swestore'))
+    for run in runs:
+        date = run.split('_')[0]
+        if misc.days_old(date) > days:
+            if dry_run:
+                LOG.info('Will remove file {} from swestore'.format(run))
+                continue
+            misc.call_external_command('irm -f {}'.format(run))
```

Comment:
You could also use [this slick function](https://github.com/NationalGenomicsInfrastructure/ngi_pipeline/blob/devel/ngi_pipeline/utils/filesystem.py#L26) that @ewels wrote to load modules from within Python